### ISSUE TITLE
Unable to generate labels on test sets for demo_folder.py

### ISSUE THREAD

**Issue by xuanyi97** [xinge008/Cylinder3D#79]
Hi, I have CUDA 10.2, spconv 1.2.1, pytorch 1.9.1 running on ubuntu18.04. Currently, I am trying to run this certain line which is;

python demo_folder.py --demo-folder YOUR_FOLDER --save-folder YOUR_SAVE_FOLDER

where I have replace YOUR_FOLDER and YOUR_SAVE_FOLDER with the necessary directory on my machine.

However, there is no labels generated and just these two lines printed:

[480 360  32]
matched parameter sets: 274, and no matched: 0

May I know what is the issue?

**Comment by xinge008** [comment]
This demo_folder is used to generate the validation results.  There is no error log. It is wired. 
Can you give some hints @ptzu

**Comment by xuanyi97** [comment]
I have moved sequence 16's Velodyne binary files into a separate folder and directed the path to that folder for demo_folder.py to generate labels using your pre-trained model. As I ran the code, the code did not carry out the for loop at line 102 as there are no variables within enumerate(demo_dataset_loader). I have checked that I provided the correct path to the sequence 16's binary files, so I am not sure why the demo_dataset_loader shows an empty list.
